test/request/accepts.js

```diff
@@ -8,8 +8,14 @@ describe('ctx.accepts(types)', function(){
     describe('when Accept is populated', function(){
       it('should return all accepted types', function(){
         const ctx = context();
-        ctx.req.headers.accept = 'application/*;q=0.2, image/jpeg;q=0.8, text/html, text/plain';
-        ctx.accepts().should.eql(['text/html', 'text/plain', 'image/jpeg', 'application/*']);
+        ctx.req.headers.accept =
```

Comment:
@stojanovic you may have misunderstood my point earlier

`ctx.accepts('text/html', 'text/plain', 'image/jpeg', 'application/*').should.equal('text/html');` is fine all in 1 line. it's just that if it's getting _too long_... then you can do something like

```javascript
ctx
  .accepts('text/html', 'text/plain', 'image/jpeg', 'application/*')
  .should.equal('text/html')
```

indenting -> chaining one function but not the others is odd.. so yea, what it currently is seems weird to me too lol @jonathanong